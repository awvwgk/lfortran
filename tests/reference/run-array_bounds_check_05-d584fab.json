{
    "basename": "run-array_bounds_check_05-d584fab",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_05.f90",
    "infile_hash": "9ef4f46c35739ba9241c8707b603e057f3ec10d97212ae6dd2ec1796",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_05-d584fab.stderr",
    "stderr_hash": "fca1454d8cb418761e6a0521d452e2f0603e321d7c260d7f3f526d5b",
    "returncode": 1
}