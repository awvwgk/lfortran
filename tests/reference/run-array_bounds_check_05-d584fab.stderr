At 7:13 of file tests/errors/array_bounds_check_05.f90
Runtime error: Array shape mismatch in subroutine 'my'

Tried to match size 3 of dimension 1 of argument number 1, but expected size is 5
